examples/with-zones/blog/next.config.js

```diff
@@ -1,6 +1,14 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   basePath: "/blog",
+  experimental: {
+    // Next.js will automatically prefix `basePath` to client side links which
+    // is useful when all links are relative to the `basePath` of this
+    // application. This option opts out of that behavior, which can be useful
+    // if you want to link outside of your zone, such as linking to
+    // "/" from "/blog" (the `basePath` for this application).
+    manualClientBasePath: true,
```

Comment:
Wouldn't it make more sense to have this as an option within the Link component?

Reply:
We are looking at making some changes related to multi zones support over the next few months, but for right now I wanted to update the examples with the current state.

Reply:
@Netail I see that you updated your comment. To add to my last message:

First, thanks for your comment! I've seen your name in other Next.js issues related to zones. If you have anything else that you'd like to share, document, or include in the example, just let me know.

For this PR: I'm also considering using `assetPrefix` in the example instead of `basePath` with `manualClientBasePath`. `basePath` with `manualClientBasePath` should work for the example, but there are things that still don't work well with `basePath` also. All that being said, I hope that this will be temporary for a better solution over the next few months.

Reply:
@mknichel Ah yes, I have. It might be good to mention that if people don't want a basePath. E.g. their app serves for different small pages. They can host their bundle on an assetPrefix, to prevent conflicting bundles

Reply:
Something like a prop "noBasePath" would be nice for the `<Link>` component, to prevent prepending the href. Quite simple to implement & would be very helpful when a basePath has been set, but you want to exclude certain `Link`s from prepending the basePath.

Reply:
I added a mention of `assetPrefix` in the README and I'll update the docs on nextjs.org as well. I'll follow up with the team about the `noBasePath` parameter to `Link` as well. Thanks!

Reply:
@mknichel muchas gracias :)